subworkflows/local/utils_nfcore_mag_pipeline/main.nf

```diff
@@ -26,7 +26,7 @@ workflow PIPELINE_INITIALISATION {
     take:
     version           // boolean: Display version and exit
     validate_params   // boolean: Boolean whether to validate parameters against the schema at runtime
-    monochrome_logs   // boolean: Do not use coloured log outputs
+    _monochrome_logs  // boolean: Do not use coloured log outputs
```

Comment:
It took me time to understand that the underscore signals that the variable is not used.  But why not just remove it in that case?

Reply:
This is something new from the nextflow formatter (in the nextflow Vscode extension). 

Yes, it's for indicating unused variables, let's say like in a map function where you try to reduce the element of a channel object (e.g. you have meta, bam, Bai, but you only need a bam) but you still need to name each component.

In this particular case however the monochrome logs from from the nf-core tempate so I don't remove it.